lib/rubocop/cop/layout/space_in_lambda_literal.rb

```diff
@@ -24,20 +24,21 @@ class SpaceInLambdaLiteral < Cop
         include RangeHelp
 
         ARROW = '->'.freeze
-        MSG_REQUIRE_SPACE = 'Use a space between `->` and opening ' \
-                            'parenthesis in lambda literals'.freeze
-        MSG_REQUIRE_NO_SPACE = 'Do not use spaces between `->` and opening ' \
-                               'parenthesis in lambda literals'.freeze
+        MSG_REQUIRE_SPACE = 'Use a space between `->` and ' \
+                            '`(` in lambda literals'.freeze
```

Comment:
Hmm, just noticed one more thing - this message is missing its final `.`. (right after the word `literals`)

Reply:
I thought we had some internal cop for checking the format of messages, but maybe this was just some idea.

Reply:
Oh, oops. I did not notice. I added `.`.

Reply:
> I thought we had some internal cop for checking the format of messages, but maybe this was just some idea.

I also thought about the same thing 😃